examples/electron-asar/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "main": "./app.js",
   "dependencies": {
```

Comment:
There is an issue with the `test-ipfs-app.app`, that is created in the electron-asar example with `npm run package`. Adding `go-ipfs-dep` and `ipfs-repo` to `examples/electron-asar/package.json` dependecies fixed it for me.
```
    "go-ipfs-dep": "0.4.13",
    "ipfs-repo": "^0.18.5",
```

Reply:
imo this is a minor issue that should not prevent merging this PR. We can fix it later.

Reply:
thanks @thisconnect!

Reply:
fixed with https://github.com/ipfs/js-ipfsd-ctl/pull/176/commits/19201d990d183fead6b27dd4b60664da995aec16